src/main/java/com/danielagapov/spawn/Controllers/EventController.java

```diff
@@ -70,21 +70,23 @@ public ResponseEntity<List<? extends IEventDTO>> getEventsCreatedByUserId(@PathV
         }
     }
 
-    // full path: /api/v1/events/friendTag/{tagId}?full=full&requestingUserId=requestingUserId
+    // TODO: path is wrong
+    // full path: /api/v1/events/friendTag/{tagId}
     @GetMapping("friendTag/{tagId}")
-    public ResponseEntity<List<? extends IEventDTO>> getEventsByFriendTagId(@PathVariable UUID tagId, @RequestParam(value="full", required=false) boolean full, @RequestParam UUID requestingUserId) {
+    public ResponseEntity<List<? extends IEventDTO>> getEventsByFriendTag(@PathVariable UUID requestingUserId, @RequestParam(required=false) UUID tagId) {
         try {
-            if (full && requestingUserId != null) {
-                return new ResponseEntity<>(eventService.convertEventsToFullFeedEvents(eventService.getEventsByFriendTagId(tagId), requestingUserId), HttpStatus.OK);
+            if (tagId != null) {
+                // needs to filter
+                return new ResponseEntity<>(eventService.getFilteredFeedEvents(requestingUserId, tagId), HttpStatus.OK);
```

Comment:
```suggestion
                return new ResponseEntity<>(eventService.getFilteredFeedEventsByFriendTagId(tagId), HttpStatus.OK);
```
sticking with our current codebase conventions